### PR TITLE
Parity validation

### PR DESCRIPTION
**Pull request by ddave2122**
Added a stand-alone file that can run parity checks for donations in our Firebase system and Stripe.  Once we know how we want to display the data, we can modify the output

### CODE REVIEW

File: donation-parity-validation.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+const {apiKey} = require('../keys/stripe.json')
```
Comment (bookjunkie93) [inline donation-parity-validation.js:1]: the keys directory is in the project root and so is this script

File: donation-parity-validation.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+const stripe = require('stripe')(apiKey);
+const serviceAccount = require('../keys/firestore-key.json');
```
Comment (bookjunkie93) [inline donation-parity-validation.js:3]: path to key should be ./keys/firestore-key.json

**Review by bookjunkie93 — CHANGES_REQUESTED** [review]
does this need to be a hotfix or can we merge into the `develop` branch first?

**Comment by ddave2122** [comment]
Base branch changed to `develop`.  Good catch!

**Review by bookjunkie93 — APPROVED** [review]

**Pull request merged**